.gitignore

```diff
@@ -21,3 +21,6 @@ roms/Pokemon - Jaune.gbc
 
 # Rust
 target
+
+# mac app
+gbmu.app
```

Comment:
it's possible to output `gbmu.app` directly into the folder `build`

Reply:
Do you mean the `target` folder ? or a different `build` folder ?

Reply:
I mean a `build` folder not `target`